src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy

```diff
@@ -30,62 +31,105 @@ class BugsnagVariantOutputTask extends DefaultTask {
-        def outputDir = processManifest.manifestOutputDirectory
+        File directoryBundle
+        List<File> manifestPaths = new ArrayList<>()
 
-        if (outputDir instanceof File) {
-            directory = outputDir
-        } else {
-            // gradle 4.7 introduced a provider API for lazy evaluation of properties,
-            // AGP subsequently changed the API from File to Provider<File>
-            // see https://docs.gradle.org/4.7/userguide/lazy_configuration.html
-            directory = outputDir.get().asFile
+        boolean getAssembleManifest = BugsnagPlugin.isRunningAssembleTask(variant, variantOutput, project)
+        boolean getBundleManifest = BugsnagPlugin.isRunningBundleTask(variant, variantOutput, project)
+
+        // If the manifest location could not be reliably determined, attempt to get both
+        if (!getAssembleManifest && !getBundleManifest) {
+            getAssembleManifest = true
+            getBundleManifest = true
         }
 
-        File manifestFile = Paths.get(directory.toString(), variantOutput.dirName, "AndroidManifest.xml").toFile()
+        def processManifest = BugsnagPlugin.resolveProcessManifest(variantOutput)
 
-        if (!manifestFile.exists()) {
-            project.logger.error("Failed to find manifest at ${manifestFile}")
-        } else {
-            project.logger.info("Found manifest at ${manifestFile}")
+        if (getAssembleManifest) {
+            def outputDir = processManifest.manifestOutputDirectory
+
+            if (outputDir instanceof File) {
+                directory = outputDir
+            } else {
+                // gradle 4.7 introduced a provider API for lazy evaluation of properties,
+                // AGP subsequently changed the API from File to Provider<File>
+                // see https://docs.gradle.org/4.7/userguide/lazy_configuration.html
+                directory = outputDir.get().asFile
+            }
+
+            File manifestFile = Paths.get(directory.toString(), variantOutput.dirName, "AndroidManifest.xml").toFile()
+
+            if (!manifestFile.exists()) {
+                project.logger.error("Failed to find manifest at ${manifestFile}")
+            } else {
+                project.logger.info("Found manifest at ${manifestFile}")
+                manifestPaths.add(manifestFile)
+            }
+        }
+
+        // Attempt to get the bundle manifest directory if required
+        if (getBundleManifest) {
+            if (!processManifest.hasProperty("bundleManifestOutputDirectory")) {
+                project.logger.error("Android Gradle plugin version 3.3.0 or higher is required to read/write the bundle manifest")
+            } else {
+                directoryBundle = processManifest.bundleManifestOutputDirectory
+                File manifestFileBundle = Paths.get(directoryBundle.toString(), variantOutput.dirName, "AndroidManifest.xml").toFile()
+                if (!manifestFileBundle.exists()) {
```

Comment:
It'd be possible to extract a function that takes `manifestFileBundle` and `manifestFile` as a parameter, and remove some duplicated code